cloud.qmd

```diff
@@ -13,7 +13,7 @@ If you want to read a single file directly into memory, you can pass a URL direc
 
 ```{r}
 #| label: read_parquet_url
-read_parquet("https://github.com/apache/arrow/raw/main/r/inst/v0.7.1.parquet")
+read_parquet(system.file("v0.7.1.parquet", package = "arrow"))
```

Comment:
We want to keep this as a URL though, right?

Reply:
Yeah, we do.

Reply:
Ah sorry, I wasn't reading, just watching the line count